src/styles/components/_specificationBlock.scss

```diff
@@ -3,6 +3,12 @@ $breakpoint-tablet: 820px;
 .cmp-specifications-list {
   display: block;
 
+  .cmp-specification-block {
+    &__heading {
+      font-size: map.get($font-sizes, "md");
+    }
+  }
```

Comment:
This isn't super BEM-y. I think what I would do here is something like this:

```scss
$breakpoint-tablet: 820px;

.cmp-specification-block {
  &__wrapper {
    display: block;

    @media only screen and (min-width: $breakpoint-tablet) {
      display: grid;
      grid-template-columns: 1fr 1fr;
    }
  }

  &__heading {
    font-size: map.get($font-sizes, "md");
  }
}
```

And then in the markup, replace "cmp-specifications-list" with "cmp-specification-block__wrapper".

Reply:
I'd also rename this file to be kebab-case for consistency.

Reply:
Those should be taken care of: https://github.com/sparkbox/accessible-components/pull/47/commits/1d8ac9a9d50cfe7fdbf4106779ca610b50f3ff35